packages/serve/src/startDevServer.ts

```diff
@@ -33,16 +39,21 @@ export default function startDevServer(compiler, devServerArgs): object[] {
     devServerOptions.forEach((devServerOpts): void => {
         const options = mergeOptions(cliOptions, devServerOpts);
         options.host = options.host || 'localhost';
```

Comment:
@ylemkimon I think we have a problem here too, I think we used `0.0.0.0` by default here, we should use the same logic as used in Node.js `http`/`https` module

Reply:
But let's solve this in other PR

Reply:
Changing the default hostname would be a breaking change for webpack-cli.

Reply:
Let's do it only for v4

Reply:
I think it's better done on the dev-server's side.

Reply:
Actually the default host was `localhost` in the webpack-dev-server CLI: https://github.com/webpack/webpack-dev-server/blob/4ab1f21bc85cc1695255c739160ad00dc14375f1/bin/options.js#L175

Reply:
Yes, but I think it is bug and we should fix it, ideally we should be like
```
const http = require('http');

const requestListener = function (req, res) {
    res.writeHead(200);
    res.end('Hello, World!');
}

const server = http.createServer(requestListener);
server.listen(8080);

console.log(server.address())
```

`::` for `IPv6`
`0.0.0.0` for `IPv4`